snyk-monitor/README.md

```diff
@@ -190,10 +190,18 @@ By default, `snyk-monitor` does not scan containers that are internal to Kuberne
 
 If you prefer to override this, you can add your own list of namespaces to exclude by setting the `excludedNamespaces` to your own list.
 For example:
-```yaml
+```shell
 --set excludedNamespaces={kube-node-lease,kube-public,local-path-storage,some_namespace}
 ```
 
+## Using custom CA certificate
+
```

Comment:
I think I need to add a sentence about why someone would add a custom CA cert? (not sure what to write)